pip/qsharp/__init__.py

```diff
@@ -10,9 +10,11 @@
     set_quantum_seed,
     set_classical_seed,
     dump_machine,
+    StateDump,
+    ShotResult,
 )
 
-from ._native import Result, Pauli, QSharpError, TargetProfile, StateDump
+from ._native import Result, Pauli, QSharpError, TargetProfile, StateDumpData
```

Comment:
Is this import used?

Reply:
Good catch, the added `StateDumpData` is not! That's a holdover from a renaming.